### ISSUE TITLE
Duplicate package: `the_silver_searcher` and `the-silver-searcher`

### ISSUE THREAD

**Issue by scw** [ContinuumIO/anaconda-issues#10133]
### Actual Behavior

Two separate entries for `ag` are listed:

https://anaconda.org/anaconda/the_silver_searcher
https://anaconda.org/anaconda/the-silver-searcher

### Expected Behavior

A single listing for The Silver Searcher, the duplication makes it confusing to know which one to install (the underscore version looks to be correct -- it has newer packages and is the same name used by other instances of the package (such as conda-forge). I don't know if there's a way to fold these packages into a single one (redirects?), but if possible it'd be nice to only have one entry for these two packages.

**Comment by msarahan** [comment]
repo.anaconda.com is the only authoritative source of packages. The anaconda channel on anaconda.org is a mirror (and a bad one that shouldn't be used, by the way, because it loses the distinction between the newer "main" defaults channel and the older "free" defaults channel.

the_silver_searcher is the only one on:

https://repo.continuum.io/pkgs/main/linux-64/repodata.json

**Comment by scw** [comment]
@msarahan I just linked to the packages as they were listed on anaconda.org, I see both when searching the normal "defaults" channel:

```
the-silver-searcher          2.0.0                         1  defaults
the_silver_searcher          2.1.0                h20f5147_0  defaults
                             2.2.0                         0  defaults
```

**Comment by mingwandroid** [comment]
Yes the defaults channel includes the old free channel but importantly at a
lower priority.

Is this duplication causing you some problems?

**Comment by msarahan** [comment]
Yeah, fair enough. I'll reopen since defaults includes both.
